tasks/create_rhaptos_site.yml

```diff
@@ -7,36 +7,58 @@
     chdir: "{{ source_dir }}/cnx-buildout"
   ignore_errors: True
 
-- name: create rhaptos database user
+- name: create temporary rhaptos database user
   when: rhaptos_site|failed
-  vars:
-    db_user: rhaptos
-    db_password: "{{ archive_db_password }}"
-  include: tasks/create_db_user.yml
+  postgresql_user:
+    name: rhaptos
+    password: "{{ archive_db_password }}"
+    login_host: "{{ archive_db_host }}"
+    port: "{{ archive_db_port }}"
```

Comment:
Wouldn't it be better to make the create_db_user task more robust rather than create a copy of it? Also, this doesn't ensure the prerequisite dependency (`psycopg2`) has been installed on the system.